deploy/helm/values.yaml

```diff
@@ -1754,36 +1754,46 @@ agamotto:
     registry: registry.cn-hangzhou.aliyuncs.com
 
 
+provider: "" # cloud be "aws","gcp","aliyun","tencentCloud", "huaweiCloud", "azure"
+validProviders:
+ - "aws"
+ - "gcp"
+ - "aliyun"
+ - "tencentCloud"
+ - "huaweiCloud"
+ - "azure"
+ - ""
 ## @section KubeBlocks default storageClass Parameters for cloud provider.
 storageClass:
   ## @param storageClass.name -- Specifies the name of the default storage class.
   ## If name is not specified and KubeBlocks deployed in a cloud, a default name will be generated.
   ##
   name: ""
-
   ## @param storageClass.create -- Specifies whether the storage class should be created. If storageClass.name is not
   ## specified or generated, this value will be ignored.
   ##
   create: true
-
   mountOptions:
     - noatime
     - nobarrier
   provider:
-    eks:
+    aws:
       volumeType: gp3
       fsType: xfs
-    gke:
+    gcp:
       volumeType: pd-balanced
       fsType: xfs
     aliyun:
       volumeType: cloud_essd
       fsType: xfs
-    aks:
+    azure:
       volumeType: managed
       fsType: xfs
-    tke:
+    tencentCloud:
       volumeType: CLOUD_SSD
+    huaweiCloud: # Huawei Cloud
+      volumeType: SAS
```

Comment:
Why use SAS ?

Reply:
> Why use SAS ?
I referenced the disk type examples from https://support.huaweicloud.com/usermanual-cce/cce_10_0380.html#section3.
Huawei Cloud seems to offer two disk types, SAS and SSD. To be honest, I'm not sure which one to use, but SAS appears to be cheaper

Reply:
Use SSD as default